wcomponents-i18n/src/main/resources/com/github/bordertech/wcomponents/messages.properties

```diff
@@ -39,6 +39,8 @@ bordertech.wcomponents.message.fieldDateAfterToday={0} must be after today.
 bordertech.wcomponents.message.fieldMinSelect={0} must have {1} or more options selected.
 bordertech.wcomponents.message.fieldMaxSelect={0} must have {1} or less options selected.
 bordertech.wcomponents.message.fieldInvalidPattern={0} is not in the required format.
+bordertech.wcomponents.message.fileWrongType=The file you have selected is not of an accepted type. Only the following type/s are accepted: %s.
+bordertech.wcomponents.message.fileWrongSize=The file you have selected is too large. Maximum file size is %s.
 
```

Comment:
@marksreeves what do you think of these validation messages. Are they fit for purpose, or too long?

Reply:
They are equivalent to the client-side messages. Length is not really a major problem in this case: better to be long and clear than brief and ambiguous or misleading.

Reply:
Thanks @marksreeves yes you are correct, I did copy the text from https://github.com/BorderTech/wcomponents/blob/5eccfb58835ea250196f8b6cccde802bd76df686/wcomponents-i18n/src/main/resources/com/github/bordertech/wcomponents/theme-messages.properties#L292